torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -30,20 +81,47 @@ std::vector<Expr*> GPULower::getLoweredExprs() {
 
   auto indexed_loops = IndexLowering::getIndexedExprs(fusion_, unrolled_loops);
 
-  return indexed_loops;
+  lowered_exprs_ = indexed_loops;
```

Comment:
we don't need the indexed_loops local variable anymore?

Reply:
we don't, but it's a good reminder when someone adds a pass that we need to explicitly set lowered_exprs.

Reply:
I mean, why not assign directly to lowered_exprs_ ?